test/compliment/sources/t_class_members.clj

```diff
@@ -207,3 +207,28 @@
 
   (fact "static class members have docs"
     (src/static-member-doc "Integer/parseInt" (-ns)) => (checker string?)))
+
+(deftest literals-inference-test
+  (fact "Has around 19 candidates (give or take, varies per JDK),
+which indicates that the members are an exact match against the class of `[]`"
+    (let [c (count (src/members-candidates "." (-ns) (ctx/cache-context
+                                                      "(__prefix__ [])")))]
+      (< 17 c 21))
```

Comment:
I propose testing for something exact. Let's say, check that when completing `.in`, only `.indexOf` is suggested when context is present, and more than one candidate are suggested without context.

Reply:
I finally did a bit of both, LMK what you think